### PR TITLE
Fixes #9216: Error message when restarting rudder-agent during relay install

### PR DESCRIPTION
**Pull request by ncharles**
https://www.rudder-project.org/redmine/issues/9216

### CODE REVIEW

File: techniques/system/common/1.0/cf-served.st [hunk 1]
```diff
@@ -105,17 +105,6 @@ bundle server access_rules
...
 
-  &if(NOVA)&
-    any::
-      "${query_types}"
-        handle => "server_access_grant_${query_types}_for_hosts",
-        comment => "Grant ${query_types} reporting query for the hub on the hosts",
-        resource_type => "query",
-        report_data_select => rudder_data_select_host,
-        admit => { ${def.policy_server}, @{sys.ip_addresses} };
-  &endif&
```
Comment (amousset) [inline techniques/system/common/1.0/cf-served.st]: Why was this code here in the first place?

Comment (amousset) [inline techniques/system/common/1.0/cf-served.st]: ping @ncharles

Comment (ncharles) [inline techniques/system/common/1.0/cf-served.st]: This is for interconecting rudder/mission-portal

Comment (ncharles) [inline techniques/system/common/1.0/cf-served.st]: this is a duplicate of l69

**Pull request merged**